test/pylib/scylla_cluster.py

```diff
@@ -1347,7 +1347,10 @@ async def _before_test(self, test_case_name: str) -> str:
         root_logger = logging.getLogger()
         # file handler file name should be consistent with topology/conftest.py:manager test_py_log_test variable
         parent_test_name = self.test_uname.replace('/', '_')
-        self.test_case_log_fh = logging.FileHandler(f"{self.base_dir}/{parent_test_name}_{test_case_name}_cluster.log")
+        self.test_case_log_fh = logging.FileHandler(f"{self.base_dir}/{parent_test_name}.{test_case_name}_cluster.log")
+        with open('/tmp/scylla_cluster', 'w') as f:
```

Comment:
how did we suddenly get "/tmp" here? Aren't we careful to put temporary files in some configurable temporary directory, not /tmp?

Reply:
Leftovers from debugging. Deleted it.